se.sjoerd.GIScan.appdata.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, appdata, desktop and icon files should come from your repo/tarball, they are not specific to Flathub.

Reply:
Thanks, I moved those files to the regular repo on my Github. Hopefully that's better!